packages/lesswrong/components/review/FrontpageReviewWidget.tsx

```diff
@@ -212,7 +218,9 @@ const FrontpageReviewPhase = ({classes}: {classes: ClassesType}) => {
   }
 
   // NB: There's nothing ensuring that unnominated20NN is a valid filter
-  const allEligiblePostsUrl = `/allPosts?timeframe=yearly&after=${REVIEW_YEAR}-01-01&before=${REVIEW_YEAR+1}-01-01&limit=100&sortedBy=top&filter=unnominated${REVIEW_YEAR}`
+  const allEligiblePostsUrl = 
+    isEAForum ? `/allPosts?timeframe=yearly&after=${REVIEW_YEAR}-01-01&before=${REVIEW_YEAR+1}-01-01&limit=100&sortedBy=top&filter=unnominated` 
+    : `/allPosts?timeframe=allTime&after=2020-01-01&before=2021-01-01&limit=100&sortedBy=top&filter=unnominated`
```

Comment:
I think yearly is actually useful. It shows you the same information, but puts the year at the top, which is helpful orientation for users who aren't used to the review process. But it's your link, your call.

Reply:
The problem is that it also weirdly truncates personal blogposts and emphasizes shortform. I agree with the year being at the top is nice.

Reply:
What if we set `displayShortform=false`?

I'll add the url query, but I'll let you decide about personal blogposts.